src/main/resources/warnings.json

```diff
@@ -673,7 +673,8 @@
     "url": "https://jenkins.io/security/advisory/2017-04-10/",
     "versions": [
       {
-        "pattern": ".*"
+        "lastVersion": "1.5.2",
+        "pattern": "1[.][012345](|[.-].*)"
```

Comment:
I thought the check is <= lastVersion using the regex, it seems I am wrong.
how about I bounce the version and release 1.6?